UnityProject/Assets/Scripts/TutorialScript/TutoPlayer.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class TutoPlayer : MonoBehaviour
+{
+    [SerializeField] private GameObject playerClothes;
+    [SerializeField] private float waitTimeCleaningClothes;
+    private PlayerScript playerScript;
+    /// Start is called before the first frame update
+    void Start()
+    {
+        //destroy itself when not on tutorial
+        if(!GameManager.Instance.onTuto)
```

Comment:
we use ``== false`` instead of ``!``. Please change all instances of the ``!`` operator for ``== false``